dspace-server-webapp/src/main/java/org/dspace/app/rest/model/BitstreamPropertiesRest.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model;
+
+public class BitstreamPropertiesRest {
```

Comment:
New class is missing Javadocs

Reply:
We will verify whether BitstreamRest can be used instead
If not, the JavaDocs will be included

Reply:
It was indeed possible to use the BitstreamRest. It has been adjusted accordingly